src/components/interfaces/HMI_API.xml

```diff
@@ -1935,9 +1935,9 @@
        True: Available, False: Not Available, Not present: Not Available.
      </description>
    </param>
-   <param name="availableHDsAvailable" type="Boolean" mandatory="false" >
```

Comment:
Same change requested as described for availableHDs above.

Reply:
@JackLivio Thanks. Fixed in be2dd5badfc9fef8734b56fce1a1827f8ac1aee0